src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -952,15 +958,8 @@ def path.map (f : G →g G') (hinj : function.injective f) {u v : V} (p : G.path
     exact hp.2 hx, },
 end⟩
 
-lemma connected_of_edge_connected {k : ℕ} (hk : 0 < k) (h : G.edge_connected k) :
-  G.connected :=
-begin
-  intros v w,
-  specialize h ∅ (by simp) (by simp [hk]) v w,
-  simp only [finset.coe_empty, subgraph.delete_edges_of_empty] at h,
-  cases h,
-  exact ⟨h.map (subgraph.map_spanning_top _)⟩,
-end
+lemma connected_of_edge_connected {k : ℕ} (hk : 0 < k) (h : G.edge_connected k) : G.connected :=
+  (h ∅ (by simp) (by simp [hk])).imp_right (by simp) -- breakpoint
```

Comment:
After the proposed changes I couldn't figure out a way to make this work because this proof is not clear to me.